Update microsite with milestone feedback (#28)
ATNoG/gsma-open-gw-apis: microsite/src/pages/documents.md

## Calendar

- [Calendar for the project*](/documents/calendar.pdf)

***Note**: This calendar can be subject to change in the future.

## Milestone 1

On this milestone, we were asked to present an overview of our project, such as the project's context, its main problems and goals, and some of the related work. Moreover, we were also required to show the project's calendar.

### Milestone's File

- [Presentation M1](/documents/Presentation%20M1.pdf)

### Feedback and Improvements

For this milestone, we received feedback from our professors highlighting areas for improvement, including poor time management, as we exceeded the allotted presentation time. To address this, we began conducting practice runs with our tutor to refine our delivery. Additionally, we were questioned about whether our project was solely focused on API implementation. In response, we emphasized that our work not only involved developing APIs but also deploying various services to ensure a comprehensive solution.

> Additionally, we also made sure to point out, in an email sent to the course's regent, that as we were working with a fairly new technology aimed at the industry, and not academic research, the quantity of **research papers** was reduced which meant we were going to use mostly **white papers** and **press releases**. Moreover, we also asked the course's regent if there would be any impact if we did not show a **graphic interface**, this because our projects was not aimed at a final client that would work with a **User Interface**, but instead developers that would integrate our solution with their product. We were told that it would not have any impact; however, we had to show the result of our work over the semester.

Reviewer: ```suggestion
Additionally, we also made sure to point out, in an email sent to the course's regent, that as we were working with a fairly new technology aimed at the industry, and not academic research, the quantity of **research papers** was reduced which meant we were going to use mostly **white papers** and **press releases**. Moreover, we also asked the course's regent if there would be any impact if we did not show a **graphic interface**, as our project is not aimed at a final client that would work with a **User Interface**, instead it is aimed at developers that would integrate our solution with their product. We were told that it would not have any impact; however, we had to show the result of our work over the semester.
```
This part can be tweaked, maybe saying target instead of repeating aimed at.
